FBSimulatorControl.xcodeproj/project.pbxproj

```diff
@@ -3400,6 +3416,10 @@
 			isa = XCBuildConfiguration;
 			baseConfigurationReference = AA7E55C31CFEC5B6009209AE /* FBDeviceControlTests.xcconfig */;
 			buildSettings = {
+				FRAMEWORK_SEARCH_PATHS = (
```

Comment:
This is good to go, but could you remove the changes to the `FRAMEWORK_SEARCH_PATHS` for the project, this isn't needed anymore as we're not using `OCMock`